app-info/eos-extra.xml

```diff
@@ -139,7 +139,7 @@
     <kudos>
       <kudo>GnomeSoftware::popular</kudo>
     </kudos>
-    <bundle type="flatpak"></bundle>
+    <bundle type="flatpak">app/org.libreoffice.LibreOffice//stable</bundle>
   </component>
   <component type="desktop" merge="append">
     <id>net.blockout.Blockout2.desktop</id>
```

Comment:
`net.blockout.BlockOutII`

Reply:
> There are many items here without `<bundle>` which come from Flatpaks. I annotated a few. 

Yeah, there are some apps which are also not popular. I think they are here because they specify additional categories for those apps. However `<bundle>` is not specific requirement for those, but it's good to have in the `<component>`

Reply:
Should be addressed by https://github.com/endlessm/gnome-software-data/pull/14